core/validatorapi/validatorapi_test.go

```diff
@@ -1626,7 +1626,7 @@ func TestComponent_SubmitValidatorRegistration(t *testing.T) {
 	allPubSharesByKey := map[core.PubKey]map[int]tbls.PublicKey{corePubKey: {shareIdx: pubkey}} // Maps self to self since not tbls
 
 	// Configure beacon mock
-	bmock, err := beaconmock.New()
+	bmock, err := beaconmock.New(beaconmock.WithGenesisTime(time.Now().Add(-time.Hour)))
```

Comment:
Why is that?

Reply:
Still need to double check why this test was failing otherwise.